interface.html

```diff
@@ -5,6 +5,7 @@
         <p>Leave empty to use Fliplet Error tracker or if you own a Sentry Project you can use your DSN.</p>
```

Comment:
Can we change this to `Error logging is enabled. Disable this add-on to stop error logging.`?